core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -143,6 +143,10 @@ private TableProperties() {
   public static final String DELETE_PARQUET_PAGE_SIZE_BYTES = "write.delete.parquet.page-size-bytes";
   public static final int PARQUET_PAGE_SIZE_BYTES_DEFAULT = 1024 * 1024; // 1 MB
 
+  public static final String PARQUET_PAGE_ROW_COUNT_LIMIT = "write.parquet.page-row-count-limit";
```

Comment:
Could this be `page-row-limit` instead of `count-limit`? I think count is probably unnecessary.

Reply:
This is done